table/block_based/block_based_table_reader_test.cc

```diff
@@ -250,6 +262,62 @@ class BlockBasedTableReaderTest
   uint32_t compression_dict_bytes_;
 };
 
+TEST_P(BlockBasedTableReaderTest, Get) {
+  Options options;
+  if (udt_enabled_) {
+    options.comparator = test::BytewiseComparatorWithU64TsWrapper();
+  }
+  options.persist_user_defined_timestamps = persist_udt_;
+  size_t ts_sz = options.comparator->timestamp_size();
+  std::vector<std::tuple<std::string, std::string>> kv =
+      BlockBasedTableReaderBaseTest::GenerateKVMap(
+          100 /* num_block */,
+          true /* mixed_with_human_readable_string_value */, ts_sz,
+          /*same_key_diff_ts=*/true);
+
+  std::string table_name = "BlockBasedTableReaderTest_Get" +
+                           CompressionTypeToString(compression_type_);
+
+  ImmutableOptions ioptions(options);
+  CreateTable(table_name, ioptions, compression_type_, kv,
+              compression_parallel_threads_, compression_dict_bytes_);
+
+  std::unique_ptr<BlockBasedTable> table;
+  FileOptions foptions;
+  foptions.use_direct_reads = use_direct_reads_;
+  InternalKeyComparator comparator(options.comparator);
+  NewBlockBasedTableReader(foptions, ioptions, comparator, table_name, &table,
+                           true /* bool prefetch_index_and_filter_in_cache */,
+                           nullptr /* status */, persist_udt_);
+
+  ReadOptions read_opts;
+  ASSERT_OK(
+      table->VerifyChecksum(read_opts, TableReaderCaller::kUserVerifyChecksum));
+
+  for (size_t i = 0; i < kv.size(); i += MultiGetContext::MAX_BATCH_SIZE) {
```

Comment:
Should this be `++i`? Seems like we currently only read 1 out of `MAX_BATCH_SIZE` keys

Reply:
I thought reading each and every value would be too lengthy. But this is just a unit test, thanks for the suggestion, I updated it to read each value.